packages/utils/test/unit/sleep.test.ts

```diff
@@ -13,20 +12,19 @@ describe("sleep", function () {
     const controller = new AbortController();
     setTimeout(() => controller.abort(), 10);
 
-    // Sleep for longer than the current test timeout.
-    // If the abort signal doesn't work mocha will throw a timeout error
```

Comment:
Comment seems relevant

Reply:
First it's not mocha any more, secondly the tests is clear that we are expecting an AbortError from the AbortController.

Reply:
> First it's not mocha any more

Yeah of course you would have to rephrase to vitest

>  clear that we are expecting an AbortError from the AbortController.

That is clear but why choose value 5000ms? seems random, sleep time only has to be sufficiently greater than timeout when abort signal is called

But not worth discussing, let's remove comment then